dts/arc/emsk.dtsi

```diff
@@ -147,6 +147,7 @@
 			compatible = "snps,designware-spi";
 			reg = <0xf0006000 0x1000>;
 			label = "SPI_0";
+			clock-frequency = <20000000>;
```

Comment:
Shouldn't it be `50000000`?
Previously it was specified in `soc/arc/snps_emsk/dts_fixup.h` by:
```
#define DT_SPI_DW_SPI_CLOCK	DT_NS16550_F0009000_CLOCK_FREQUENCY
```
and `clock-frequency` in `uart@f0009000` is set to: `<DT_APB_CLK_HZ>`, and this one is `50000000`.

Actually, I'm not sure if the above definition of `DT_SPI_DW_SPI_CLOCK` is used or rather the one from `soc/arc/snps_emsk/soc.h` but the latter also leads to the value of `MHZ(50)`.

Reply:
Fixed, could somebody knowing the hardware confirm correct setting?